plugins/auth-backend/package.json

```diff
@@ -43,6 +43,7 @@
     "@backstage/plugin-auth-backend-module-gitlab-provider": "workspace:^",
     "@backstage/plugin-auth-backend-module-google-provider": "workspace:^",
     "@backstage/plugin-auth-backend-module-oauth2-provider": "workspace:^",
+    "@backstage/plugin-auth-backend-module-okta-provider": "workspace:^",
```

Comment:
It seems that the "_@davidzemon/passport-okta-oauth_" is no longer used. Could you check if there is any other dependency cleanup we can do?

Reply:
removed `@davidzemon/passport-okta-oauth` from auth-backend